chromadb/types.py

```diff
@@ -72,7 +72,7 @@ def __init__(
         self,
         id: UUID,
         name: str,
-        configuration: CollectionConfiguration,
+        configuration: CollectionConfigurationInternal,
```

Comment:
should the user-facing Collection.py view expose the configuration too?

Reply:
I think the only reason for this would be to copy the settings in one configuration to create a collection with the same settings. Otherwise, Configuration only contains immutable things from when the collection is created, so I've chosen not to expose it to the user on the Collection object.